.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,44 @@
+name: Github Actions Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-18.04
+    name: Unit tests
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12.22.1
+      - name: Cache build directories
```

Comment:
We don't need to cache the build directories if we don't have another job in the workflow that needs to use them

Reply:
Good stop! updated.